rootToH5converter/Event.py

```diff
@@ -25,13 +26,20 @@ def __init__(self, data_processor, i_event, delta_r, use_fat_jets=False):
         self.metEta = data_processor.get_value_from_tree("MET_eta", i_event, i_object)
         
         self.nJets = data_processor.get_value_from_tree("N_fat_jets" if use_fat_jets else "N_jets", i_event)
-        self.nTracks = data_processor.get_value_from_tree("N_tracks", i_event)
+        if input_type == InputTypes.PFnanoAOD102X:
+            if use_fat_jets:
+                N_tracks_variable = "N_tracks_AK8"
+            else:
+                N_tracks_variable = "N_tracks_AK4"
+        else:
+            N_tracks_variable = "N_tracks"
+        self.nTracks = data_processor.get_value_from_tree(N_tracks_variable, i_event)
         self.nNeutralHadrons = data_processor.get_value_from_tree("N_neutral_hadrons", i_event)
         self.nPhotons = data_processor.get_value_from_tree("N_photons", i_event)
         
         # load tracks from tree
         self.tracks = []
-        self.fill_tracks()
+        self.fill_tracks(input_type, use_fat_jets)
```

Comment:
I would suggest to store input type in self.input_type rather than passing it around as an argument.

Reply:
Yes I can change that